hw/ip/edn/data/edn_testplan.hjson

```diff
@@ -5,29 +5,27 @@
   name: "edn"
   import_testplans: ["hw/dv/tools/dvsim/testplans/csr_testplan.hjson",
                      "hw/dv/tools/dvsim/testplans/intr_test_testplan.hjson",
+                     "hw/dv/tools/dvsim/testplans/alert_test_testplan.hjson",
                      "hw/dv/tools/dvsim/testplans/tl_device_access_types_testplan.hjson",
                      "hw/dv/tools/dvsim/testplans/stress_all_with_reset_testplan.hjson"]
   entries: [
     {
       name: smoke
       desc: '''
-            Smoke test acessing a major datapath within the edn.
-
-            **Stimulus**:
-            - TBD
-
-            **Checks**:
-            - TBD
+            Enable edn, let edn boot mode generate csrng instantiate/generate commands,
+	    endpoint requests genbits, compare csrng/endpoint genbits.
```

Comment:
Verify that the software port works properly, polling each command that gets issued for ack.

Reply:
Doesn't the bullet "Verify SW_CMD_REQ/SW_CMD_STS registers/bits behave as predicted" cover this?